src/app/layouts/global/Header.tsx

```diff
@@ -1,133 +1,55 @@
-    }));
-    // navigate("/");
-    // try {
-    //   void login({
-    //     password: "",
-    //     username: "",
-    //   }).unwrap();
-    // } catch (error) {
-    //   console.log({
-    //     error,
-    //   });
-    // }
-  };
-  return (
-    <div>
-      <button type="button" onClick={loginHandler}>Login</button>
-      <button type="button" onClick={logoutHandler}>Logout</button>
-    </div>
-  );
-}
-
-function LanguageToggle (): JSX.Element {
+export default function Header (): JSX.Element {
   const t = useTypedTranslation();
-
-  const handleChangeLanguage = (event: MouseEvent<HTMLButtonElement>): void => {
-    const { name } = event.target as HTMLButtonElement;
-    void changeLanguage(name);
-  };
+  const [search, setSearch] = useState("");
 
   return (
-    <>
-      {(["en", "uk"] as TranslationKeys[]).map((language, index) => (
-        <button
-          key={language}
-          name={language}
-          onClick={handleChangeLanguage}
+    <header>
+      <div className="flex justify-between relative z-10 bg-white px-[5%] items-center">
+        <Logo/>
+        <RoleLink/>
+        <LanguageToggle/>
+      </div>
+      <svg className="relative z-0 -top-[0.5rem]" width="100%" viewBox="0 0 20 0.7" style={{ filter: "drop-shadow(0px 4px 20px rgba(50, 64, 161, 0.5))" }}>
+        <path
+          d="M 0 0.4 C 4.5 0.8 5 0.3 9 0.3 C 14 0.3 15.4 0.8 20 0.4 L 20 0 L 0 0"
+          fill="white"
         >
-          {t(language)}
-          {index === 0 ? "|" : ""}
-        </button>),
-      )}
-    </>
-
-  );
-}
-
-const RoleLink = ({ role }: { role: RoleUnion; }): JSX.Element => {
-  const t = useTypedTranslation();
-  switch (role) {
-    case MANAGER: {
-      return <Link to={"/catalog-for-manager"}>{t("catalog")}</Link>;
-    }
-    case ADMIN: {
-      return <Link to={"/catalog-for-admin"}>{t("catalog")}</Link>;
-    }
-    default: {
-      return (
-        <>
-          <Link to={"/about-us"}>{t("aboutAs")}</Link>
-          <Link to={"/contacts"}>{t("contacts")}</Link>
-        </>
-      );
-    }
-  }
-};
-
-export default function Header (): JSX.Element {
-  const role = useAppSelector(state => state.user.role);
-  return (
-    <header className="flex justify-between p-3 bg-[#D9D9D9]">
-      <p>Logo</p>
-      <div className="flex">
-        <div>
-          <LanguageToggle/>
-          <RoleLink role={role}/>
-          <LogButtons/>
+        </path>
+      </svg>
+      <div className="flex justify-between items-center px-[5%] mt-4">
+        <button className="flex items-center catalog-icon">
+          <Catalog/>
+          <p className="pl-1 h6">{t("catalog")}</p>
+        </button>
+        <div className="search-icon w-[500px] p-1 flex items-center border-black border rounded-md">
+          <Search/>
+          <input className="p-1 grow outline-none" placeholder={t("search")} type="text" name="search" value={search} onChange={(e) => setSearch(e.target.value)}/>
+          {search &&
```

Comment:
This is not good idea to use && for conditional render. Better approach is using ternary operator.
Read more: https://dev.to/aishanipach/dont-use-for-conditional-reasoning-3525